src/main/resources/application.properties

```diff
@@ -1 +1,20 @@
-messages.file=messages.txt
+# Properties for the sample tracking service implementation
```

Comment:
Should there be default values here? I'd argue that either the values can be retrieved from the set environment variables or the reporter should stop

Reply:
The app will fail with the default values anyhow